families/sawtooth_validator_registry/validator_registry/src/validator_registry_tp_verifier.rs

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright 2018 Intel Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -----------------------------------------------------------------------------
+ */
+
+extern crate openssl;
+
+use self::openssl::{
+    hash::MessageDigest,
+    pkey::{PKey, Public},
+    sha::sha256,
+    sign::Verifier,
+};
+use crypto::{digest::Digest, sha2::Sha256};
+use protos::validator_registry::{SignUpInfo, SignUpInfoProof, ValidatorRegistryPayload};
+use sawtooth_sdk::{
+    messages::setting::Setting,
+    processor::handler::{ApplyError, TransactionContext},
+};
+
+use validator_registry_tp_handler::{get_state, parse_from};
+
+const _CONFIG_ADDRESS_PART_SIZE: usize = 16;
```

Comment:
These should be renamed SETTINGS_ADDRESS_PART_SIZE,  SETTING_MAX_KEY_PARTS, and SETTING_NAMESPACE

Config is a holdover from the first name of settings

Reply:
This legacy name should be updated - it is a hold-over from the 0.8 version of the validator.